src/models/Inputs.ts

```diff
@@ -1,5 +1,9 @@
 interface Inputs {
     truckSize: 'small' | 'medium' | 'large';
+    depreciationPeriod?: number;
+    kilometersPerDay?: number[];
+    daysPerWeek?: number;
+    hoursPerDay?: number;
```

Comment:
Shall we make them non-nullable and set default values in `useState` (which we already do)? It is easier to avoid the null checks later in output calculations.